spec/jobs/pusher_notification_job_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe PusherNotificationJob, '#perform' do
+  subject { described_class.new.perform(0, appointment) }
+
+  context 'when the appointment is for today' do
+    let(:appointment) { build_stubbed(:appointment, start_at: Time.zone.today) }
```

Comment:
`Time.zone.today` returns as date while start_at should be a  datetime. Can we use `Time.zone.now` instead